src/components/Icons/src/index.tsx

```diff
@@ -89,6 +90,7 @@ export const TrashIcon: F = (props: SvgIconProps) => buildIcon(TrashSvg, props);
 export const AlertTriangleIcon: F = (props: SvgIconProps) => buildIcon(AlertTriangleSvg, props);
```

Comment:
Why are these single letter types, what is an F?

Reply:
line 62 of `Icons/src/index.tsx`: `type F = React.FunctionComponent<SvgIconProps>; // less typing`